### DISCUSSION TITLE
Server instantly closes after loading world if port set is in use elsewhere

### DISCUSSION THREAD

**Discussion by Wintar** [Pryaxis/TShock#2544] [Q&A] [answered]
OS: Windows 10, 64-bit
TShock Version: 4.5.12
[Log from an attempt with port set to 13155](https://github.com/Pryaxis/TShock/files/7775211/2021-12-24_14-40-12.log)
[Log from an attempt with default settings](https://github.com/Pryaxis/TShock/files/7775212/2021-12-24_14-41-55.log)
[ServerLog.txt](https://github.com/Pryaxis/TShock/files/7775215/ServerLog.txt)
No debug log is made despite having them enabled.
[Video of what happens](https://drive.google.com/uc?id=1zOHAiIlz_4Zq-PrBg0kX4cBxJ4NpOqaF)

Not sure if this is my fault or not. I tried several other ports: 7777, 7778, 23155, 11111, and 77777. All worked, except the last, which just re-prompted me. 13155 is the only port that fails, and is the only port that I've forwarded for this computer.

Also, this is a fresh download. I haven't done anything to TShock's files yet, and no other plugins are added. The only file not from the latest release is a .rar of my previous version's files. Running as an administrator didn't change anything.

**EDIT:** I tried a different forwarded port of 27015, and it worked fine. I forwarded 7777 and used defaults, and it worked fine. I unforwarded 13155 and tried it, and it still closes. Why is specifically 13155 causing it to close? I know I can just forward a different port, but I usually use 13155 for everything, so I was hoping to not have to change it.

**Comment by Wintar** [comment, marked as answer]
As it turns out, this was entirely my fault, luckily. The port was being used elsewhere. The only minor issue is no explanation from TShock, but it's really still my own failure. I would delete this embarrassingly silly discussion if I could.

**Comment by drunderscore** [comment]
Yeah, unfortunately Terraria will allow you to start the server on the same port, and everything will seem fine until it just exits.
